src/sentry/utils/email.py

```diff
@@ -350,16 +351,34 @@ def send(self, to=None, bcc=None, fail_silently=False):
 
     def send_async(self, to=None, bcc=None):
         from sentry.tasks.email import send_email
+        # from sentry/emails/send-invite.txt to send-invite
         messages = self.get_built_messages(to, bcc=bcc)
         for message in messages:
-            safe_execute(send_email.delay, message=message,
-                         _with_transaction=False)
+            safe_execute(
+                send_email.delay,
+                message=message,
+                _with_transaction=False,
+            )
+            logger.info(
+                name='sentry.mail',
+                event='mail.queued',
+                to=to,  # WIP: truncating on human fmt and multi-line for machine.
```

Comment:
I was thinking that this logic could be done inside the log handler itself just generically for all lists passed through.